frontend/src/components/smart/EarningsCalculator.vue

```diff
@@ -168,6 +170,19 @@ export default Vue.extend({
     isLoadingCharacter(): boolean {
       return !this.currentCharacter;
     },
+
+    currentTokenPrice(): number {
```

Comment:
we might need a bit in the readme about "how to add a new network" that includes "add it to app-config.json, then add a curenttokenprice getter", etc